@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -189,6 +189,7 @@ export class TextElementsRenderer {
 
     private m_tmpVector = new THREE.Vector2();
     private m_overloaded: boolean = false;
+    private m_catalogsLoading: number = 0;
```

Comment:
Why is it assumed that loading a catalog will trigger render calls next frame? What if you get a 404? What if it takes 2sec to load the catalog?

Reply:
> Why is it assumed that loading a catalog will trigger render calls next frame

Sorry in that line I was mislead with the title of the PR, I understood that it referred to something else.

As it happens, by chance the other questions are still relevant I think!